ansible/roles/adm/tasks/main.yml

```diff
@@ -39,3 +39,17 @@
 
 - name: sudoers.d/adm
   template: src=sudoers dest=/etc/sudoers.d/adm owner=root group=root mode=0440 validate='visudo -cf %s'
+
+- name: lockdown access to ssh
+  lineinfile: dest=/etc/ssh/sshd_config
+              regexp="^PasswordAuthentication"
+              line="PasswordAuthentication no"
+              state=present
+  notify: restart ssh
+
+- name: do not allow root to login via ssh
+  lineinfile: dest=/etc/ssh/sshd_config
+              regexp="^PermitRootLogin"
+              line="PermitRootLogin no"
+              state=present
+  notify: restart ssh
```

Comment:
IMHO, that's bad idea to drop `root` login in `adm` role as it's played while logging in as root :)
I suggest to keep `adm` role doing nothing but creation of administrator's accounts and move various defaults to separate role.